src/ac_training_lab/cobot280pi/server.py

```diff
@@ -0,0 +1,246 @@
+from my_secrets import *
```

Comment:
Not a huge deal, but in general maybe avoid import * patterns

Reply:
fixed in latest push